maintenance/idr0021-data-prep.md

```diff
@@ -0,0 +1,90 @@
+
+Workshop data preparation (idr0021)
+===================================
+
+This document details the steps to prepare data from IDR for a workshop demonstrating
+analysis with Fiji, usage of Map Annotations and OMERO.tables and filtering with OMERO.parade.
+We use IDR0021, which is a Project containing 10 Datasets with a total of ~400 Images.
+
+
+Download IDR data
+=================
+
+External users
+--------------
+
+Please contact the OME team for access to download IDR data.
+
+Internal users
+--------------
+
+Follow instructions on the
+[IDR submission workflow](https://docs.google.com/document/d/1TmBZ43_yhiO3AOua8oMk4mPWKWJtpeYNc2KLP17h-1I/edit#) to download the data via ```ssh``` from ```idr0-slot3```.
+
+
+Prepare IDR-metadata and import
+===============================
+
+Clone https://github.com/IDR/idr-metadata and edit
+```idr-metadata/idr0021-lawo-pericentriolarmaterial/experimentA/idr0021-experimentA-filePaths.tsv```
+to point ALL paths at the location of the data downloaded above.
+e.g.
+```Dataset:name:CDK5RAP2-C	/full/path/to/data/idr0021/CDK5RAP2-C/Centrin_PCNT_Cep215_20110506/Centrin_PCNT_Cep215_20110506_Fri-1545_0_SIR_PRJ.dv```
```

Comment:
I do not suppose you did it manually. It has over 400 lines. Do you have the script you used please ? Maybe you could please upload it as a PR to the IDR-metadata repo ? (or to the [new idr0021... repo](https://github.com/sbesson/idr0021-lawo-pericentriolarmaterial) established by @sbesson ?)

the script could be there in a subfolder ``scripts`` as it is for example on https://github.com/IDR/idr0041-cai-mitoticatlas/tree/master/scripts ?